config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

```diff
@@ -282,7 +282,7 @@ periodics:
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
 
-- name: periodic-cluster-api-e2e-workload-upgrade-1-24-1-25-main
+- name: periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-main
```

Comment:
We have to keep 1-24-1-25 and add a new job for 1-25-1-26
(same for all the branches, main, 1.2, and also 1.3)

Reply:
Got it!

So, now we have `1-24-1-25`, `1-25-1-26`. What about the `1-25-latest`, we need to change it `1-26-latest` right?

Reply:
The end state must be: `1-24-1-25`, `1-25-1-26`, `1-26-latest` (that is equivale to testing upgrades to all the stable releases + the upgrade to latest)